src/styles/_catalog.scss

```
@@ -1,22 +1,32 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(1, 200px);
```

no nee to repeat 1 column
```suggestion
  grid-template-columns: 200px;
```